### DISCUSSION TITLE
get number of row in partition of nvs flash

### DISCUSSION THREAD

**Discussion by TechnoElectro** [Mair/esp32-course#146] [Course Qeustions]
Hello
I need to know how many records are in my partition table.
So I can get the last 7, or get the latest data from my partition.
what is the best practice?

**Comment by Mair** [comment]
hi @TechnoElectro, Not sure what you mean by "get the last 7"
you can get the partition table with 
`idf.py partition_table`

**Reply by TechnoElectro** [reply]
thanks for your reply
If we call partition table (DataBase table), then every Structured Data that we insert in NVS has an id number(row number) .
`sprintf(idNum, "id_%d", *id);`
I want to know how many records are in my Database table. or what is the last row number?
